non-critical-infra/modules/mailserver/default.nix

```
@@ -9,8 +9,29 @@
 
     fqdn = config.networking.fqdn;
 
-    # TODO: change to `nixos.org` when ready
-    domains = [ "mail-test.nixos.org" ];
+    domains = [ "nixos.org" ];
+  };
+
+  sops.secrets."nixos.org.mail.key" = {
+    owner = "opendkim";
+    group = "opendkim";
+    mode = "0600";
+
+    # How to generate:
+    #
+    # ```console
+    # cd non-critical-infra
+    # DOMAIN=nixos.org
+    # SELECTOR=mail
+    # PRIVATE_KEY_PATH=secrets/$DOMAIN.$SELECTOR.key.umbriel
+    # nix shell nixpkgs#opendkim --command opendkim-genkey --selector="$SELECTOR" --domain="$DOMAIN" --bits=1024
+    # mv mail.private "$PRIVATE_KEY_PATH"
+    # sops encrypt --in-place "$PRIVATE_KEY_PATH"
+    # ```
+    #
+    # Next, look at `mail.txt` and update DNS accordingly.
+    sopsFile = ../../secrets/nixos.org.mail.key.umbriel;
+    path = "/var/dkim/nixos.org.mail.key";
```

I think this part is missing:

```suggestion
    path = "/var/dkim/nixos.org.mail.key";
    format = "binary";
```

---

Yep, sorry. Shouldn't have pushed before my local build had finished :blush: